src/main/java/com/inventoryservice/model/Category.java

```diff
@@ -0,0 +1,42 @@
+package com.inventoryservice.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "category")
+@Data
```

Comment:
use @Getet @setter @EqualHashCode

Reply:
completed and updated code by using @Getter and @Setter instead of @Data.